### PR TITLE
Make it possible to set coinbase to address as argument.

### PR DESCRIPTION
**Pull request by rantan**
This PR is for issue #3 .

### CODE REVIEW

File: src/command_args.rs [hunk 1]
```diff
@@ -636,6 +670,7 @@ fn test_invalid_private_key() {
         matches,
         config: Some(ConfigToml {
             signer: Some(SignerToml {
+                to_address: Some("mbaffffff".to_string()),
```
Comment (azuchi) [inline src/command_args.rs]: This test spec is a test for `privatekey: Some ("aabbccdd" .to_string ())`, so couldn't verify the error of to_address?

Comment (rantan) [reply src/command_args.rs]: Yes, my change's intention is just path the compile. but so that, it should be `None`. I'll change it.

Comment (azuchi) [reply src/command_args.rs]: In addition to it, test for address format check also need.

Comment (rantan) [reply src/command_args.rs]: Yes, i also add it.

Comment (rantan) [reply src/command_args.rs]: I've added 2 commits for it.

**Pull request merged**